ts/twitter/src/app/login/_components/account-modal.tsx

```diff
@@ -1,23 +1,103 @@
 "use client";
 
-import { Button } from "@chakra-ui/react";
+import theme from "@/lib/theme";
+import { Button, Divider, Input, Text, VStack } from "@chakra-ui/react";
+import { useState } from "react";
 import type React from "react";
+import { FaApple, FaGoogle } from "react-icons/fa";
 
 interface AccountModalProps {
-  onNext: () => void;
+  onNext: (username: string) => void;
```

Comment:
```suggestion
  handleNextButtonClick: (username: string) => void;
```
Let's rename the event handler to align with the naming conventions in [the React official doc](https://react.dev/learn/responding-to-events).